src/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.csproj

```diff
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <!-- We are no longer shipping this assembly and it is only used in desktop facades.
+        The reference assembly version needs to be freezed as desktop facades require an exact version. -->
```

Comment:
```suggestion
        The reference assembly version needs to be frozen so that our desktop compat facades point to the version from the package. -->
```